app/controllers/dealerships_controller.rb

```diff
@@ -1,7 +1,7 @@
 class DealershipsController < ApplicationController
 
   def index
-    @dealerships = Dealership.all
+    @dealerships = Dealership.order('created_at DESC').all
```

Comment:
I see the logic for sorting here. Just want to make sure this kind of sorting doesn't more properly belong to a model